config.go

```diff
@@ -71,13 +71,13 @@ func (mc *MTLSConfig) ParseMTLSFiles() (tls.Certificate, *x509.CertPool, error)
 func (mc *MTLSConfig) validateRequiredMTLSParams() error {
```

Comment:
I think we should be able to replace this function with the `required` validation tags, no?

Reply:
agree, but we need that for handling this custom error here https://github.com/conduitio-labs/conduit-connector-grpc-server/blob/feat/sdk-updates/config.go#L52.

Reply:
Do you mean the multi-error [here](https://github.com/conduitio-labs/conduit-connector-grpc-server/blob/feat/sdk-updates/config.go#L82)? Because it's just a joined error from the required errors.

Reply:
No, i mean in ParseMTLSFiles() we are returning an [error](https://github.com/conduitio-labs/conduit-connector-grpc-server/blob/feat/sdk-updates/config.go#L52) suggesting the user to disable mTLS secure connection. 
If we put the required validation then it will don't give this suggestion.

Reply:
To be honest, I think we can remove that error message, because we have it documented in the README. It was handy earlier when the connector was validating the configs itself, but now most of that work is done by the SDK.

Reply:
Then should i remove Disable(bool) field also from the config? Since if we put the required validation for those fields, if the field is not provided, it will throw error, even if we put disable as true.

Reply:
Ah, the `Disabled` field is what I was missing. Everything makes sense now, thanks for the clarification!